src/main/python/hydra/rmqtest/runtest.py

```diff
@@ -64,9 +64,9 @@ def rerun_test(self, options):
         l.info("PUB server updated")
 
         # Create test groups
-        g1 = self.create_app_group(self.rmqsub, "test-group", apps_in_group=10, analyser=HAnalyser)
-        g2 = self.create_app_group(self.rmqsub, "test-group2", apps_in_group=5, analyser=HAnalyser)
-        g3 = self.create_app_group(self.rmqsub, "test-group3", apps_in_group=5, analyser=HAnalyser)
+        g1 = self.create_app_instances_group(self.rmqsub, "test-group", num_app_instances=10, analyser=HAnalyser)
```

Comment:
Looks like the name change is going to break other stuff, let's revert the name change.